front/app/components/HookForm/QuillMultilocWithLocaleSwitcher/index.tsx

```diff
@@ -35,10 +40,13 @@ const QuillMultilocWithLocaleSwitcher = ({ name, ...rest }: Props) => {
   );
 
   // Select the first error messages from the field's multiloc validation error
-  const errorMessage = Object.values(
+  const validationError = Object.values(
     (errors[name] as Record<Locale, FieldError> | undefined) || {}
   )[0]?.message;
 
+  const apiError =
```

Comment:
Do I read this right in that it evaluates to a boolean? If so, I'd suggest naming it `hasApiError` so it's clear it's true or false

If not, maybe add a comment as to what it's supposed to evaluate to, because I missed it 😅